index.html

```diff
@@ -871,6 +892,32 @@ <h3>Presentations</h3>
 
     </section>
 
+    <section class="informative">
+      <h3>Bound Credentials and Unbound Credentials</h3>
+
+      <p>
+        There are 2 broad categories of <a>verifiable credentials</a>: <a>bound credentials</a> and <a>unbound credentials</a>. 
+        <a>Bound credentials</a> are bound a specific credential <a>subject</a>. 
+        <a>Unbound credentials</a> are not bound to a <a>subject</a>. 
+        As such, <a>unbound credentials</a> cannot be used to create <a>verifiable presentations</a> -
+        they are simply "data that has been signed by an <a>Issuer</a> and held by any <a>Holder</a>".
+        <a>Unbound credentials</a> are easily issued to and exchanged between groups of two or more <a>Holders</a>. 
+        The most common use case for <a>unbound credentials</a> is to represent all types of business documents like invoices and purchase orders 
+        (that are not simple extensions or representations of a single subject).
+      </p>
+      <p>
+        <a>Bound credentials</a> contain a <a>subject</a> id element inside the credentialSubject element 
+      (which binds the credential to a subject and enables the <a>Holder</a> of the <a>credentential</a> 
+      to derive <a>verifiable presentations</a> from the <a>bound credential</a>).
+      </p>
+      <p>
+        <a>Unbound credentials</a> do not contain a <a>subject</a> id element inside the credentialSubject element.
+        An <a>unbound credential</a> is "like signed data issued to a <a>Holder</a>". 
+        A specific <a>unbound credential</a> can be held by 2 more <a>Holders</a> at the same time. For example, an invoice
```

Comment:
This is true for any sort of VC, not just unbound credentials. Any credential can be held by more than one Holder at any given time. Some VCs can be bound to the `credentialSubject.id` at presentation time. This language needs to be tightened up so that it won't confuse people reading the specification.